dragonfly_schema/_openapi.py

```diff
@@ -81,6 +81,26 @@ def get_openapi(
         }
         tags.append(tag)
 
+        
+        # sort properties order: put required parameters at begining of the list
+        s = schemas[name]
+        if not 'required' in s:
```

Comment:
This is on me but this line should actually be:

```suggestion
        if 'required' not in s:
```

Both will work.